### PR TITLE
ATLAS-203: Report server errors to the user

### PR DESCRIPTION
**Pull request by HelioStrike**
JIRA Issue: https://issues.openmrs.org/browse/ATLAS-203

@bmamlin I created a constants.js file because there 30-40 lines logging database errors! If this is ok, perhaps we could copy in other constants from the codebase in a future PR?

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+module.exports = {
+
+    DATABASE_ERROR_RESPONSE: "An unexpected database error occurred. Please ask an administrator to check atlas logs at YYYY-DD-MM HH:MM:SS for more information"
```
Comment (bmamlin) [inline constants.js]: This is great; however, I've missed where "YYYY-DD-MM HH:MM:SS" is being converted to the server's current timestamp. Is that happening somewhere? Or are we sending this message unchanged?

Comment (HelioStrike) [reply constants.js]: Distant screaming: NOOOOOOOOOOOOOOO!!!!!!

Updated the PR :)))

Comment (bmamlin) [reply constants.js]: I hate to nitpick, but you've got dozens of cookie-cutter lines that require specific knowledge of the constant's format, do fancy slicing & dicing of dates, and require `util` and `constants.js`. Instead, how about eliminating `constants.js` and, instead, adding this to our `util.js`?

```node
genericDatabaseErrorMessage: function() {
	return util.format(
		"An unexpected database error occurred. Please ask an administrator to check atlas logs at %s for more information",
		dateformat(new Date(), 'mm/dd/yyyy h:MM:ss TT')
	)
}
```

Then you only need to require our `util.js` and the entries can go from this:

```node
return res.status(500).send({ message: util.format(constants.DATABASE_ERROR_RESPONSE, new Date().toISOString().slice(0, 19).replace('T', ' ')) });
```

to this:

```node
return res.status(500).send(util.genericDatabaseErrorMessage());
```

Comment (HelioStrike) [reply constants.js]: Much better! Updated the PR :)

**Pull request merged**